test/test_heuristics.rb

```diff
@@ -61,6 +61,13 @@ def test_detect_still_works_if_nothing_matches
     assert_equal Language["Objective-C"], match
   end
 
+  def test_1to9_by_heuristics
+    assert_heuristics({
+      "Roff" => all_fixtures("Roff", "*.{1..9}"),
```

Comment:
Your test is failing because `*.{1..9}` isn't a valid glob pattern for a set:

```
irb(main):001:0> Dir.glob("./samples/Roff/*.{1..9}")
=> []
```

```suggestion
      "Roff" => all_fixtures("Roff", "*.[1-9]"),
```

... should do the trick.

```
irb(main):002:0> Dir.glob("./samples/Roff/*.[1-9]")
=> ["./samples/Roff/as.1", "./samples/Roff/fstat.2", "./samples/Roff/moo.6", "./samples/Roff/ino.8", "./samples/Roff/cat.4", "./samples/Roff/sample.4", "./samples/Roff/vs.9", "./samples/Roff/pow.3", "./samples/Roff/core.5", "./samples/Roff/greek.7", "./samples/Roff/soelim.8"]
irb(main):003:0>
```

Same below.

Reply:
> *.{1..9} isn't a valid glob pattern for a set:

Ugggh, I always get `*.[1-9]` confused with `1..2`. 😰 Blame `git log`'s syntax for showing a range of commits.

I was about to ask, *"how did this not fail the first time I submitted this?"* but noticed the assertion responsible for this was only [added recently](https://github.com/github/linguist/pull/4410/files#diff-952df3fccfc8a6df1579b4554d15c6b8R23). Will update.